network-api/networkapi/buyersguide/templates/bg_base.html

```diff
@@ -93,7 +93,7 @@
       {% block guts %}{% endblock %}
     </div>
 
-    {% include "partials/footer.html" with exclude_language_switcher=True %}
```

Comment:
do we still use `exclude_language_switcher` anywhere else? If not, we should clean up the footer partial, too.

Reply:
Yes, it’s used on 404 pages